sdk/identity/azpki/cloud_config.go

```diff
@@ -0,0 +1,41 @@
+//go:build go1.18
+// +build go1.18
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+package azpki
+
+import "github.com/Azure/azure-sdk-for-go/sdk/azcore/cloud"
+
+// Cloud Service Names for Monitor Query Logs and Metrics, used to identify the respective cloud.ServiceConfiguration
+const (
+	ServiceNameLogs cloud.ServiceName = "azpkiLogs"
```

Comment:
The const name and value should be specific to your SDK ("Logs" is from the logging service of `azquery`).
```go
ServiceNamePKI cloud.ServiceName = "pki"
```